Source/FakeItEasy.Specs/UnconfiguredFakeSpecs.cs

```diff
@@ -6,23 +6,28 @@
 
     public static class UnconfiguredFakeSpecs
     {
+        public interface IFoo
+        {
+            bool IsADummy { get; }
+        }
+
         [Scenario]
         public static void VirtualMethod(
             MakesVirtualCallInConstructor fake)
         {
-            "when faking a class with a virtual method"
+            "When faking a class with a virtual method"
                 .x(() => fake = A.Fake<MakesVirtualCallInConstructor>());
 
-            "it should return a default value when the method is called during the constructor"
+            "It should return a default value when the method is called during the constructor"

```

Comment:
Actually, I struggle with this when some of the steps appear to be implicit.

* Given an unconfigured fake that calls a virtual method during its constructor
* _I don't know that I have a `When`_.
* Then the method will return a default value
* And the method call will be recorded

* Given an unconfigured fake
* When I call a virtual method
* Then the method will return a default value
* And the method call will be recorded
?